trento/xml/article_sap_trento.xml

```diff
@@ -1097,8 +1121,38 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
               <listitem>
                 <para><guimenu>Hosts details</guimenu> section: it shows the
                   status of both the &t.agent; and the Node Exporter and
-                  provides the host name, the cluster name (when applicable) and
-                  the &t.agent; version. </para>
+                  provides the host name, the cluster name (when applicable),
+                  the &t.agent; version and the host IP addresses. </para>
+              </listitem>
+              <listitem>
+                <para>saptune summary section: saptune is a solution that comes with SLES for SAP
+                 and allows SAP admins to ensure that their SAP host are properly configured to run the corresponding SAP workloads.
+                 The integration of saptune in the Trento console gives the SAP admin visibility over the tool even when they are not
+                 working at operating system level. The integration supports saptune 3.1.0 and higher, and includes the addition of
+                 the host tuning status in the aggregated health status of the host.</para>
+                     <figure xml:id="fig-saptune-summary-section">
+        <title>saptune summary section</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata width="80%" fileref="saptune-summary-section.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+                 <para>If an SAP workload is running on the host but no saptune or a version lower than 3.1.0 is installed, a warning
+                 is added to the aggregated health status of the host. When saptune version 3.1.0 or higher, is installed, a details view
+                 is also available showing detailed information about the saptune status:</para>
+                       <figure xml:id="fig-saptune-details-view">
```

Comment:
Just a friendly warning. I usually try to avoid adding version information as much as possible. They get outdated very quickly and becomes a maintenance nightmare. Imagine you have many of these texts. You would then have to check each one to see if it is still up to date.

In this case, it may be justified. Probably it's not possible, but could you rephrase this paragraph? If not, leave it as is.

Reply:
saptune version 3.1.0 is a reference here. We cannot remove it.